### PR TITLE
add height and width attributes to html rendering

### PR DESCRIPTION
**Pull request by easterday**
fixed tests and updated readme

### CODE REVIEW

**Review by DeadWisdom — CHANGES_REQUESTED** [review]
Please add a test for this in tests/test_html.py

Thanks!

File: delta/html.py [hunk 1]
```diff
@@ -199,6 +199,11 @@ def classes_check(op):
 def image(root, op):
     el = sub_element(root, 'img')
     el.attrib['src'] = op['insert']['image']
+    attrs = op.get('attributes', None)
+    if attrs and attrs.get('width', None):
+        el.attrib['width'] = op['attributes']['width']
+    if attrs and attrs.get('height', None):
+        el.attrib['height'] = op['attributes']['width']
```
Comment (DeadWisdom) [inline delta/html.py]: No tests means you missed that height is being assigned the 'width' value.

**Review by DeadWisdom — CHANGES_REQUESTED** [review]

**Review by DeadWisdom — APPROVED** [review]
Awesome, thanks

**Comment by easterday** [comment]
thanks!

**Pull request merged**